### DISCUSSION TITLE
Need details on plan for replication from on-premise machine babelfish + postgreSQL to AWS Aurora cloud Babelfish +PostgreSQL

### DISCUSSION THREAD

**Discussion by psrinivasa2** [babelfish-for-postgresql/babelfish-for-postgresql#32] [Q&A]
Need to understand replication steps from general on-premise machine babelfish + postgreSQL to AWS Aurora cloud Babelfish +PostgreSQL

**Comment by suprio-amzn** [comment]
You can explore PostgreSQL native logical replication: https://docs.aws.amazon.com/AmazonRDS/latest/AuroraUserGuide/AuroraPostgreSQL.Replication.Logical.html.

The version numbers from where it is supported for Babelfish is mentioned in the above page.

**Comment by suprio-amzn** [comment]
The other option is AWS DMS: https://docs.aws.amazon.com/dms/latest/userguide/CHAP_Source.html
